contrib/pkg/createcluster/create.go

```diff
@@ -375,14 +369,8 @@ OpenShift Installer publishes all the services of the cluster like API server an
 	flags.StringVar(&opt.OpenStackIngressFloatingIP, "openstack-ingress-floating-ip", "", "Floating IP address to use for cluster's Ingress service")
 
 	// vSphere flags
-	flags.StringVar(&opt.VSphereVCenter, "vsphere-vcenter", "", "Domain name or IP address of the vCenter")
```

Comment:
The vsphere parameters are deleted but the vsphere sample for hiveutil create-cluster not updated accoringly, see the "case 3 - hiveutil" part of OCP-84265.

Reply:
@jianping-shu I've updated the documentation.  Please let me know if you have any questions, or need a sample `platform-spec.json`

Reply:
@jianping-shu it's my mistake: The environment variable should contain the contents of the json, not a path to the json file itself.  I've updated the docs to match the correct intended usage